src/components/Card.svelte

```diff
@@ -1,9 +1,18 @@
 <script lang="ts">
+	import { onMount } from 'svelte';
 	import type { SongDetails } from '../types/Details';
 	import Controls from './Controls.svelte';
 	export let songDetails: SongDetails,
-		expand = false;
+	expand = false;
+	let trackPosition = 0;
+	let liked = false;
+	let added = false;
+	let audio: HTMLAudioElement;
 
+	//temp audio for testing
+	onMount(() => {
+		audio = new Audio("https://p.scdn.co/mp3-preview/ce4a01f9dc6091951d0a94b640b625b46e0efbd2?cid=774b29d4f13844c495f206cafdad9c86");
```

Comment:
Don't need to hard code this to test. Can create a `Card` component in your tests and pass in the link from there.

Reply:
Are you suggesting the audio element moved from `Card` component to `Controls` component?

Reply:
1. why is there a hardcoded link?
2. consider adding a `previewAudio` property to type `songDetails`